website/addons/wiki/views.py

```diff
@@ -422,6 +428,12 @@ def project_wiki_rename(auth, wname, **kwargs):
     else:
         sharejs_uuid = wiki_utils.get_sharejs_uuid(node, new_wiki_name)
         wiki_utils.broadcast_to_sharejs('redirect', sharejs_uuid, node, new_wiki_name)
+    if auth.user.comments_viewed_timestamp.get(node._id, None) and \
```

Comment:
@sloria Do you think here I should also add <code>del wiki_page_timestamp</code>?